packages/apm/src/span.ts

```diff
@@ -81,6 +81,17 @@ export class Span implements SpanInterface, SpanContext {
    */
   public readonly startTimestamp: number = timestampWithMs();
 
+  /**
+   * Internal start time tracked with a monotonic clock.
+   *
+   * Works with mostly any browser version released since 2012.
+   * https://caniuse.com/#search=performance.now
+   *
+   * Works with Node.js v8.5.0 or higher.
+   * https://nodejs.org/api/perf_hooks.html#perf_hooks_performance_now
```

Comment:
Couldn't find what platform versions we support. What are we targeting?
I see [a Travis job with Node 8, namely v8.17.0](https://travis-ci.com/getsentry/sentry-javascript/jobs/288922495). 
I also see [Node 6, v6.17.1](https://travis-ci.com/getsentry/sentry-javascript/jobs/288922496). Likely no `performance.now` there.

What about browser support?